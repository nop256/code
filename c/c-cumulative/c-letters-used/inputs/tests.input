The quick brown FOX jumps over the lazy dog.
tHE QUICK BROWN fox JUMPS OVER THE LAZY DOG.
The "quick" fox "jumps" over the little "dog".
Seven slimy snakes slowly slithered southward sipping soup.
How much wood would a woodchuck chuck if a woodchuck could chuck wood?
hOW MUCH WOOD WOULD A WOODCHUCK CHUCK IF A WOODCHUCK COULD CHUCK WOOD?
Hello darkness, my old friend, I've come to talk with you again.
Call me Ishmael
"It was the best of times, it was the worst of times, it was the age of wisdom, it was the age of foolishness, it was the epoch of belief, it was the epoch of incredulity, it was the season of Light, it was the season of Darkness, it was the spring of hope, it was the winter of despair, we had everything before us, we had nothing before us, we were all going direct to Heaven, we were all going direct the other way..."
“Alexei Fyodorovich Karamazov was the third son of a landowner from our district, Fyodor Pavlovich Karamazov, well known in his own day (and still remembered among mus) because of his dark and tragic death, which happened exactly thirteen years ago and I which I shall speak of in its proper place.”
Lolita, light of my life, fire of my loins. My sin, my soul. Lo-lee-ta: the tip of the tongue taking a trip of three steps down the palate to tap, at three, on the teeth. Lo. Lee. Ta. She was Lo, plain Lo, in the morning, standing four feet ten in one sock. She was Lola in slacks. She was Dolly at school. She was Dolores on the dotted line. But in my arms she was always Lolita.
When Mr Bilbo Baggins of Bag End announced that he would shortly be celebrating his eleventyifirst birthday with a party of special magnificence, there was much talk and excitement in Hobbiton.
