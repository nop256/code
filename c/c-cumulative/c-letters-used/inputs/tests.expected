abcdefghijklmnopqrstuvwxyz
abcdefghijklmnopqrstuvwxyz
cdefghijklmopqrstuvx
adeghiklmnoprstuvwy
acdfhiklmouw
acdfhiklmouw
acdefghiklmnorstuvwy
acehilms
abcdefghiklmnoprstuvwy
abcdefghiklmnoprstuvwxyz
abcdefghiklmnoprstuwy
abcdefghiklmnoprstuvwxy
